hedera-mirror-rosetta/app/persistance/postgres/block/block.go

```diff
@@ -10,6 +10,11 @@ import (
 
 const genesisPreviousHash = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
 
+const (
+	whereClauseGenesisBlock string = "consensus_start = (SELECT MIN(consensus_start) FROM %s)"
```

Comment:
I think that a better way to get the genesis block is to look for parent hash -> 0000000...
My thinking is that if for some reason the db does not contain some block records (don't know if its possible, but still), we will return the earliest block (based on the consensus start prop).
However, if we are checking for `prevhash` 000 -> we will be sure that this is the genesis block.